modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -1994,10 +1995,15 @@ private void generateFilesMetadata(List<File> files) {
                         relativePath = relativePath.replace(File.separator, "/");
                     }
                     if (!relativePath.equals(relativeMeta)) {
-                        sb.append(relativePath).append(System.lineSeparator());
+                        relativePaths.add(relativePath);
                     }
                 });
 
+                Collections.sort(relativePaths, (a, b) -> IOCase.SENSITIVE.checkCompareTo(a,b));
```

Comment:
sort the relative path strings, after they've been made platform-independent in https://github.com/OpenAPITools/openapi-generator/blob/849b3e8e175759063e6c1233335ab329327cb0ad/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java#L1994